snuba/query/dsl.py

```diff
@@ -182,6 +182,10 @@ def divide(
     return FunctionCall(alias, "divide", (lhs, rhs))
 
 
+def abs(lhs: Expression, alias: Optional[str] = None) -> FunctionCall:
```

Comment:
I don't think you need this

Reply:
Do you mean I should emit `FunctionCall("abs", ...)` directly into the place where I'm using this?

Reply:
Indeed, thank you!